### PR TITLE
chore(deps)!: update to Angular 17

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

**Comment by arturovt** [comment]
@just-jeb updated dependencies to `rc` versions, but at least we can see the CI result. Will only have to update dependencies to `17.0.0` after the release is done.

**Comment by alaaahmedmu** [comment]
any updates when will this PR get merged after upgrading to v17?

**Comment by arturovt** [comment]
I kindly ask you not to spam here. Repeatedly leaving comments in this issue for us to publish faster is not helpful or necessary. Please do not repeatedly mention us if we don't reply as fast as you'd like us to.

**Comment by alaaahmedmu** [comment]
Sorry @arturovt but this is the first question. Please do not consider it as a spam. I kindly ask for something.

**Comment by just-jeb** [comment]
Hey guys, sorry for the delay, a lot is going on here is Israel. We'll publish the new version soon.

File: packages/jest/package.json [hunk 1]
```diff
@@ -42,17 +42,17 @@
     "ci": "./scripts/ci.sh"
   },
   "dependencies": {
-    "@angular-devkit/architect": ">=0.1600.0 < 0.1700.0",
-    "@angular-devkit/core": "^16.0.0",
+    "@angular-devkit/architect": ">=0.1700.0-rc.2 < 0.1800.0",
+    "@angular-devkit/core": "^17.0.0-rc.2",
     "jest-preset-angular": "13.1.2",
```
Comment (mshima) [inline packages/jest/package.json:47]: Official support landed in 13.1.3.
```suggestion
    "jest-preset-angular": "13.1.4",
```

**Review by just-jeb — APPROVED** [review]

**Pull request merged**